lib/usageParams.js

```diff
@@ -53,6 +53,11 @@ export default yargs
     type: 'number',
     default: 1000
   })
+  .option('management-host', {
+    describe: 'Management API host',
+    type: 'string',
+    default: 'api.contentful.com'
+  })
```

Comment:
You missed out adding it to the `defaultOpts` in the index.js

I guess `error-log-file` is also missing over there